dockerfiles/docker-compose.yml.template

```diff
@@ -0,0 +1,10 @@
+elasticsearch:
+  image: mozilla/dxr_elasticsearch
+  net: host
+  
+dxr:
+  image: mozilla/dxr_application
+  net: host
+  links:
```

Comment:
What was `links` for? Everything seems to work fine without for me (in my own version of this that draws liberally from yours).